tests/importer/test_importer.py

```diff
@@ -1,8 +1,21 @@
+from contextlib import contextmanager
+import os
+from pathlib import Path
+
 import pytest
 
 from uvicorn.importer import ImportFromStringError, import_from_string
 
 
+@contextmanager
```

Comment:
Please use tmp_path from pytest.

Reply:
If we write the code to a tmp_path then we also need to do something like monkeypatch-ing the PYTHONPATH env var so that the importer knows the tmp_path is somewhere to look for python.

Reply:
Then the previous code was fine. (Sorry)

Reply:
could add a mark to adjust the base tmp dir for this test? I think that would work.